src/module/actor/creature/sheet.ts

```diff
@@ -53,6 +53,14 @@ abstract class CreatureSheetPF2e<TActor extends CreaturePF2e> extends ActorSheet
                 remainingDying: Math.max(actor.attributes.dying.max - actor.attributes.dying.value),
                 remainingWounded: Math.max(actor.attributes.wounded.max - actor.attributes.wounded.value),
             },
+            specialResources: Object.values(this.actor.synthetics.resources)
+                .filter((r) => !r.itemUUID)
+                .map((r) => ({
```

Comment:
R.pick!

Reply:
I suspect eventually we'll have something there that can't simply be picked, and its why I leaned on that at the start....but I suppose I definitely should convert that now.